uber/models.py

```diff
@@ -367,13 +376,14 @@ def __getattr__(self, name):
             return suffixed
 
         try:
-            [multi] = [col for col in self.__table__.columns if isinstance(col.type, MultiChoice)]
```

Comment:
This would only ever work for classes that only have a single multi choice column. Which I suppose is the expected behavior, because we have unit tests that check for that. The code I changed this to should be much faster, though.